client/src/reducers/reducers_notification.js

```diff
@@ -0,0 +1,41 @@
+import {
+  CREATE_DOCUMENT,
+  UPDATE_DOCUMENT,
+  DELETE_DOCUMENT,
+  CRUD_DOC_ERROR
+} from '../constants/types_document_action';
+
+import {
+  SIGNED_UP,
+  SIGNUP_FAILED,
+  LOGIN_FAILED,
+  LOGGED_OUT,
+  LOGOUT_FAILED
+} from '../constants/types_account_action';
+
+export default function(state = "", action){
+  switch(action.type){

```

Comment:
would it be possible to just check whether action.payload.msg or action.payload.error exists, and then return whichever one does exist,  instead of having a case for each

Reply:
It is definitely possible, but it will potential cause future errors in the event that the developer doesn't know which action is taken place.

Reply:
The standard procedure is to have a switch-case scenario is this file because this file only serves to update elements in the reducers. There shouldn't be any kind of complex logic in this file because that's suppose to be for the actions js files.

Reply:
I'll implement it now, since it'll be a less of a headache in the future.